fvwm/icons.c

```diff
@@ -787,7 +787,7 @@ void CreateIconWindow(FvwmWindow *fw, int def_x, int def_y)
 			GetUnusedModifiers(), Scr.FvwmCursors[CRS_DEFAULT],
 			True);
 		xwc.sibling = FW_W_FRAME(fw);
-		xwc.stack_mode = Below;
+		xwc.stack_mode = Above;
```

Comment:
This isn't quite right -- we can do better with compositors here by looking at the `VisibilityNotify` property of the icon, and changing it to `Above` in such cases.  But this change by itself shouldn't go in.

Reply:
Visibility has nothing to do with it.  I screwed up not including pictures, obviously; so I'll pay the piper for a second.

To avoid the possibility of one sibling casting a shadow on the other, either they need to be processed as a single widow, or they need to be ordered in the stack to ensure one does not shadow the other.   Since Fvwm doesn't know the direction or presence of shadows for any window, it can't know how to stack windows to avoid undesired results.  To get a complete solution, Fvwm or compositor needs to aggregate the windows; or Fvwm needs to know the direction of shadow,  OR Fvwm can just use the most popular case and wait for a really long time to see it is a problem for some users.  I myself have been using Compton for nearly 5 years and never left the unaffected group, below, until recently.

I don't feel like it's a good return on investment to put icons and their titles in a containing window, though that is the right thing.  Fvwm's icon transparency is a bigger fish by far (and harder to fix.)  Most Fvwm users either don't use desktop icons, or don't title them, or don't use compositors, or keep shadows for icons off, or (microscopic group) use shadows casting up the screen.  All those users can't see a problem.  

Notice that microscopic group of up-shadow users?  Those users are affected by this change.  Fvwm uses down-right shadows unless you work to change it, so it is actively working against it's own defaults by allowing up-shadows to work better than down-shadows.  Thus changing a completely arbitrary setting to a more correct guess for those using ALL those settings above except up-shadows is correct regardless of whether it is a complete solution.  Also, there is a distinct possibility no current users would be negatively affected.

OK, here are the pics.  Note my background is a slideshow, so these are actually the same windows -- it takes a second to switch Fvwm's. 
With the change:
![withit](https://user-images.githubusercontent.com/21070892/40062801-032874bc-582a-11e8-8c82-d0b193c98867.jpg)
And without it:
![withoutit](https://user-images.githubusercontent.com/21070892/40062829-105629b8-582a-11e8-9068-4f5ce278b22e.jpg)

That was all tooo much, so congrats if you got this far.  
Close this request as either incorporated or not and I will delete the fork to save resources.   We delete forks after pull request resolution, right?